Use Multibranch pipeline job type in guided tour (#1047)
SAP/jenkins-library: documentation/docs/guidedtour.md

1. Provide a name for your new item (for example, *My First Pipeline*) and select **Multibranch Pipeline**.
   <p align="center">
   ![Create Pipeline Job](images/JenkinsNewItemPipeline.png "Jenkins New Item")
   </p>

>    **Note:** The ready-made Continuous Delivery pipelines of project "Piper" require to run as **Multibranch Pipeline**.

Reviewer: ```suggestion
   **Note:** The ready-made continuous delivery pipelines of project "Piper" must run as **Multibranch Pipeline**.
```

> 1. For **Branch Sources**, choose **Add source**, and select the **Git** as source repository.

Reviewer: "the **Git** repository as source" oder "**Git** as source repository" (ohne "and")

   <p align="center">
   ![Create Pipeline Job](images/JenkinsNewItemPipeline-AddSource.png "Branch Sources - Add source")
   </p>

1. For **Project Repository** in the **Git** section, enter the URL of your Git repository, for example `https://github.com/<your-org>/cloud-cf-helloworld-nodejs`. **Note:** If your repository is protected, you must provide your credentials in **Credentials**.

> 1. For **Discover branches**, choose **Add** and select **Filter by name (with wildcards)**.

Reviewer: Klickt man auf **Filter by name (with wildcards)**? Dann würde ich "choose **Add** and **Filter by name (with wildcards)**" schreiben.

   <p align="center">
   ![Create Pipeline Job](images/JenkinsNewItemPipeline-DiscoverBranch.png "Discover branches - Add")
   </p>

>    While a multibranch pipeline can execute different Jenkinsfiles for different branches, here, we will configure the pipeline of a single branch only. 

Reviewer: ```suggestion
   A multibranch pipeline can execute different Jenkinsfiles for different branches. In this case, however, configure the pipeline of a single branch only. 
```

1. For **Include** in the **Filter by name** section, enter the branch name `1_REST_persist_in_Memory`.
   <p align="center">
   ![Create Pipeline Job](images/JenkinsNewItemPipeline-FilterByName.png "Discover Branches - Filter By Name")
   </p>

> 1. Choose **Save**. **Result:** Jenkins scans the repository for branches and filters them according the specified **Includes**. If the branch is detected, it will be build afterwards.

Reviewer: ```suggestion
1. Choose **Save**. **Result:** Jenkins scans the repository for branches and filters them according to the specified **Includes**. If the branch is detected, it is built.
```

> For additional information about multibranch pipelines please refer to the [Jenkins documentation][jenkins-io-multibranch].

Reviewer: ```suggestion
For additional information about multibranch pipelines, please refer to the [Jenkins documentation][jenkins-io-multibranch].
```

## What's Next

You are now familiar with the basics of using project "Piper". Through the concept of pipeline as code, project "Piper" and Jenkins pipelines are extremely powerful. While Jenkins pipelines offer a full set of common programming features, project "Piper" adds SAP-specific flavors. Have a look at the different **Scenarios**  to understand how to easily integrate SAP systems with defaults. 

> Dive into the ready-made Continuous Delivery pipelines: the **General Purpose Pipeline**

Reviewer: ```suggestion
Dive into the ready-made continuous delivery pipelines: the **General Purpose Pipeline**
```

> and **SAP Cloud SDK Pipeline** help you to quickly build and deliver your apps.

Reviewer: ```suggestion
and **SAP Cloud SDK Pipeline** help you quickly build and deliver your apps.
```
